lldb/test/API/macosx/simulator/TestSimulatorPlatform.py

```diff
@@ -198,7 +202,7 @@ def test_ios_backdeploy_apple_silicon(self):
         self.run_with(
             arch=self.getArchitecture(),
             os="ios",
-            vers="11.0",
+            vers="14.0",
```

Comment:
The version 11 here is (I believe) significant. Prior to 12 there was no `x86_64-apple-ios-simulator` triple, instead simulator binaries used load commands that produced a `x86_64-apple-ios` triple, which was assumed to mean simulator, due to the combination of `x86` and `iOS`.